pages/status.tsx

```diff
@@ -181,12 +181,19 @@ const Status: FC = () => {
         </>
       ) : (
         <form onSubmit={handleFormikSubmit} id="form-get-status">
-          <p>{t('header-messages.fill-in-field')}</p>
           <p>
             <strong>{t('header-messages.matches')}</strong>
           </p>
-          <p>{t('header-messages.for-child')}</p>
-          <p>{t('header-messages.passport-officer')}</p>
+          <ul className="space-y-2 pl-4 mb-3 list-inside list-disc">
+            <li>{t('header-messages.list.item-1')}</li>
+            <li>{t('header-messages.list.item-2')}</li>
+            <li>{t('header-messages.list.item-3')}</li>
+            <li>{t('header-messages.list.item-4')}</li>
+          </ul>
+          <p>
```

Comment:
Based on the workitem I think the paragraph should be wrapped with a blockquotes like this:

```tsx
<blockquote className="py-3 px-6 mb-3 border-l-6 border-gray-200">
    <p className="m-0">
        ...
    </p>
</blockquote>
```